### PR TITLE
Add initial testing framework

### PR DESCRIPTION
**Pull request by shu8**
This PR adds an initial testing framework for `XRPackage` and `XRPackageEngine` by @yahengsu @nicoledanuwidjaja @shu8.

- Adds [AVA](https://github.com/avajs/ava) as a dev dependency
  - and AVA config in root as `ava.config.js`
- Sets up GitHub Actions to run `npm test`
- Adds tests for:
  - getting screenshots @shu8
  - handling missing icons @nicoledanuwidjaja 
  - wearing packages as avatars @yahengsu 

It's set up as follows:

- all test logic in new `tests/` folder
- the entire repo is served at `localhost:[some available port]` with express (`tests/utils/_withPageAndStaticServer.js`)
- a new `tests/static/` folder with some assets and an `index.html` that is dynamically rendered to `import {XRPackage, XRPackageEngine} from http://localhost:[port]` when requested
- individual tests are in `tests/` which use AVA and (usually) headless Puppeteer navigating to `http://localhost:[port]/tests/static` to perform the tests in the browser.

If this seems like we're going in the right direction then we could hopefully merge this with `master` before we all continue and write our individual tests.

As part of https://github.com/webaverse/xrpackage/issues/64.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -216,6 +216,27 @@ This will download the given package id locally.
...
+
+const withPageAndStaticServer = require('./utils/_withPageAndStaticServer');
+
+test('your test', withPageAndStaticServer, async (t, page) => {
+  // Your test logic, using `t.context.staticUrl`
+  t.pass();
+});
+```
+
+Note that `withPageAndStaticServer` sets the test's context `t.context.staticUrl` to be the base URL for the static server, e.g. `localhost:64880` who's root is [`./tests/static`](./tests/static).
```
Comment (avaer) [inline README.md:238]: whose?

Comment (shu8) [reply README.md:238]: Ah, my bad, I'll fix that in the next PR! :)

Comment (shu8) [reply README.md:238]: Fixed in https://github.com/webaverse/xrpackage/pull/102

Comment (avaer) [reply README.md:238]: Thanks!

**Pull request merged**